src/bundle/Resources/public/js/OnlineEditor/toolbars/config/base-table.js

```diff
@@ -1,18 +1,18 @@
-import AlloyEditor from 'alloyeditor';
-import EzConfigButtonsBase from './base-buttons';
+import EzConfigBase from './base';
 
-export default class EzConfigTableBase extends EzConfigButtonsBase {
+export default class EzConfigTableBase extends EzConfigBase {
```

Comment:
I'm not sure but `EzConfigFixedBase` could be better because as I understand this fix will work for `td` and it's at the top of the `td`. When user will use elements path and will select `table` there will be the same problem. With `EzConfigFixedBase` it will be fixed at the top o the screen if it should be outside.

Also, the second question is do we want to cover the table with toolbar (I know there was some clients that don't like when toolbar is covering what they wrote in previous line), but this is a question to @SylvainGuittard.

Reply:
I just tried the steps to reproduce and I understand the pain point here. 
I think it makes sense for the table toolbar to be visible by the user.

Reply:
Hi guys,
Adding the UX perspective here. I would go for a similar solution that the RTE Paragraph toolbar has right now, given this behavior was added after being user tested, I agree @dew326.
Maybe a potential option to make this work better would be also to remove the _fixed_ position from `ez-edit-header__content-type-name`.